src/win/tty.c

```diff
@@ -62,6 +63,9 @@
 #define ENABLE_VIRTUAL_TERMINAL_PROCESSING 0x0004
 #endif
 
+#define UNDERLINE_CURSOR      25
```

Comment:
It's not obvious that this value is a cursor size. Maybe call it `CURSOR_SIZE_UNDERLINE` ?